deployments/helm/dra-example-driver/templates/_helpers.tpl

```diff
@@ -95,3 +95,15 @@ Create the name of the service account to use
 {{- default "default" .Values.serviceAccount.name }}
 {{- end }}
 {{- end }}
+
+{{/*
+Create the name of the service account to use for the webhook
+*/}}
+{{- define "dra-example-driver.webhookServiceAccountName" -}}
+{{- $name := printf "%s-webhook-service-account" (include "dra-example-driver.fullname" .) }}
+{{- if .Values.webhook.serviceAccount.create }}
+{{- default $name .Values.webhook.serviceAccount.name }}
+{{- else }}
+{{- default "default-webhook" .Values.webhook.serviceAccount.name }}
+{{- end }}
+{{- end }}
```

Comment:
I opted to create a separate ServiceAccount for the webhook since the webhook doesn't require any RBAC permissions. The Helm chart already seemed to be assuming that there will be only one ServiceAccount though, so the values structure is a little clunky where the kubelet plugin's ServiceAccount is rooted at `serviceAccount` where the webhook's is rooted at `webhook.serviceAccount`. Happy to make that more consistent how if that's desired.